components/server/ee/src/prebuilds/github-service.ts

```diff
@@ -32,8 +32,10 @@ export class GitHubService extends RepositoryService {
             return <ProviderRepository>{
                 name: r.name,
                 cloneUrl: r.clone_url,
-                account: r.owner?.login,
-                accountAvatarUrl: r.owner?.avatar_url,
+                account: r.owner.login,
+                accountAvatarUrl: r.owner.gravatar_id
+                    ? `https://www.gravatar.com/avatar/${r.owner.gravatar_id}?size=128`
```

Comment:
**issue**: This is still failing to load the avatar when there's no gravatar set, right?

| New project | Project branches |
|-|-|
| <img width="466" alt="Screenshot 2022-03-15 at 8 00 55 PM" src="https://user-images.githubusercontent.com/120486/158442429-e8b572c8-40c3-4114-b0cb-387d8730d1a0.png"> | <img width="349" alt="Screenshot 2022-03-15 at 8 00 43 PM" src="https://user-images.githubusercontent.com/120486/158442425-1785f750-2e35-4b15-ac7b-e15dae226e17.png"> |

Reply:
Ah, I see.